### PR TITLE
remove-containerd-workaround

### PR DESCRIPTION
**Pull request by calvix**
remove workaround fro regsitry container auth
towards https://github.com/giantswarm/roadmap/issues/1737

### CODE REVIEW

File: helm/cluster-aws/templates/_helpers.tpl [hunk 1]
```diff
@@ -95,24 +95,16 @@ room for such suffix.
 - systemctl restart kubelet
 {{- end -}}
 {{- define "registryFiles" -}}
-{{- if and .Values.registry .Values.registry.configure -}}
-- path: /opt/registry-config.toml
+{{- if .Values.registry }}
+{{- if .Values.registry.configure -}}
+- path: /opt//etc/containerd/conf.d/registry-config.toml
```
Comment (AverageMarcus) [inline helm/cluster-aws/templates/_helpers.tpl:100]:
```suggestion
- path: /opt/etc/containerd/conf.d/registry-config.toml
```

File: helm/cluster-aws/templates/_helpers.tpl [hunk 2]
```diff
@@ -95,24 +95,16 @@ room for such suffix.
 - systemctl restart kubelet
 {{- end -}}
 {{- define "registryFiles" -}}
-{{- if and .Values.registry .Values.registry.configure -}}
-- path: /opt/registry-config.toml
+{{- if .Values.registry }}
+{{- if .Values.registry.configure -}}
```
Comment (AverageMarcus) [inline helm/cluster-aws/templates/_helpers.tpl:99]: Are we missing an `{{ end }}` ? We've added two if's but no extra end.

Comment (AverageMarcus) [reply helm/cluster-aws/templates/_helpers.tpl:99]: Looks like just an issue with the PR view. Ignore :)

**Review by AverageMarcus — APPROVED** [review]

File: helm/cluster-aws/templates/_helpers.tpl (hunk 2 above)
Comment (calvix) [reply helm/cluster-aws/templates/_helpers.tpl:99]: the diff is tricky as the next block si removed but one `{{end}}` is left

**Comment by calvix** [comment]
cluster looks happy

**Pull request merged**